src/site/content/en/secure/same-site-same-origin/index.md

```diff
@@ -149,26 +174,28 @@ have a different eTLD+1 are "cross-site".
   </table>
 </div>
 
-### "schemeful same-site"
+### "schemeless same-site"
 
-{% Img src="image/admin/Y9LbVyxYzg4k6mwSEqyE.png", alt="schemeful same-site", width="677", height="105" %}
+{%
+  Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/G2VfM8BA5Q5LTaz22Puu.png",
+  alt="schemeless same-site", width="800", height="123"
+%}
 
-The definition of "same-site" is evolving to consider the URL scheme as part of
-the site in order to prevent HTTP being used as [a weak
+The definition of "same-site" evolved to consider the URL scheme as part of the
+site in order to prevent HTTP being used as [a weak
 channel](https://tools.ietf.org/html/draft-west-cookie-incrementalism-01#page-8).
-As browsers move to this interpretation you may see references to "scheme-less
-same-site" when referring to the older definition and "[schemeful
-same-site](/schemeful-samesite/)" referring to the stricter definition. In that
-case, `http://www.example.com` and `https://www.example.com` are considered
-cross-site because the schemes don't match.
+On the contrary, the older concept of "same-site" without scheme comparison is
+now called "schemeless same-site". For example, `http://www.example.com` and
+`https://www.example.com` are considered schemelses same-site but not same-site,
+because the eTLD+1 part only matters and the scheme is not taken into account.
 
 <div class="table-wrapper">
   <table>
     <thead>
       <tr>
         <th>Origin A</th>
         <th>Origin B</th>
-        <th>Explanation of whether Origin A and B are "schemeful same-site"</th>
+        <th>Explanation of whether Origin A and B are "schemeless same-site"</th>
```

Comment:
So these were just same-site before? I thought schemeful can be mapped to same-site and schemeless is a new concept?

Reply:
Yes and yes. I don't see anything contradicting here? What's wrong?

Reply:
I guess my question is in the phrasing of the header: the header indicates an explanation of whether or not Origin A and B are schemeful same-site. If we're mapping like for like, it'd read "Explanation of whether Origin A and B are "same-site"" wouldn't it?

Reply:
So this change is to reflect the spec update: what was called "schemeful same-site" is now "same-site", and "same-site" is now "schemeless same-site". Clarified with @anusmitaray offline.